lib/templates/exportFile.js

```diff
@@ -17,7 +17,8 @@ class ExportFile {
 
   /**
    * Process the response provided by the Silverfin API and store every detail in its corresponding file (liquid files, config file, etc)
-   * @param {number} firmId
+   * @param {string} type firm or partner
+   * @param {number} envId  The id of the firm or partner environment where the template is going to be imported from
    * @param {object} template
    */
   static save(type, envId, template) {
```

Comment:
This is probably why you removed the `await` clauses.

This may be something to monitor an align, now `ReconciliationText` and `SharedParts` have `async save` while `ExportFile` and `AccountTemplate` doesn't :thinking: